docs-src/src/components/BodyConfigCombat.jsx

```diff
@@ -173,6 +173,23 @@ class BodyConfigCombat extends PureComponent {
     this.setState({ combatLBASGroups: value })
   }
 
+  handleCombatForceRetreatNodeAdd = (node) => {
```

Comment:
This is a lot of code for what could basically be replaced with the Multi Select component (see the Expeditions dropdown multiselects).